failed build in maven docker image

### Maven Version

3.9.5

### Version

0.0.5

### Sentry SDK Version

6.0.0

### Steps to Reproduce

Dockerfile
```Dockerfile
FROM maven:3-amazoncorretto-11-debian as builder
WORKDIR /root
ADD . .
ENV SENTRY_AUTH_TOKEN="***"
RUN --mount=type=cache,target=/root/.m2 mvn clean install -Pprod -Dmaven.test.skip=true
```
mvn.pom
```xml
...
            <plugin>
                <groupId>io.sentry</groupId>
                <artifactId>sentry-maven-plugin</artifactId>
                <version>0.0.5</version>
                <extensions>true</extensions>
                <configuration>
                    <skip>${sentry.skip}</skip>
                    <!-- for showing output of sentry-cli -->
                    <debugSentryCli>true</debugSentryCli>

                    <org>sentry</org>

                    <project>live-backend</project>

                    <!-- in case you're self hosting, provide the URL here -->
                    <!--                    <url>https://sentry.backend-manage.com/</url>-->

                    <!-- provide your auth token via SENTRY_AUTH_TOKEN environment variable -->
                    <authToken>${env.SENTRY_AUTH_TOKEN}</authToken>
                </configuration>
                <executions>
                    <execution>
                        <goals>
                            <!--
                            Generates a JVM (Java, Kotlin, etc.) source bundle and uploads your source code to Sentry.
                            This enables source context, allowing you to see your source
                            code as part of your stack traces in Sentry.
                            -->
                            <goal>uploadSourceBundle</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
...
```

### Expected Result

can build success

### Actual Result

```shell
...
6.331 [INFO] ----------------------< com.live:live-client-api >----------------------
6.331 [INFO] Building live-client-api 0.0.1-SNAPSHOT                            [3/4]
6.331 [INFO]   from live-client-api/pom.xml
6.331 [INFO] --------------------------------[ jar ]---------------------------------
6.336 [WARNING] Parameter 'parameters' is unknown for plugin 'maven-compiler-plugin:3.6.1:compile (default-compile)'
6.336 [WARNING] Parameter 'parameters' is unknown for plugin 'maven-compiler-plugin:3.6.1:compile (default-compile)'
6.336 [WARNING] Parameter 'parameters' is unknown for plugin 'maven-compiler-plugin:3.6.1:testCompile (default-testCompile)'
6.377 [WARNING] The artifact mysql:mysql-connector-java:jar:8.0.33 has been relocated to com.mysql:mysql-connector-j:jar:8.0.33: MySQL Connector/J artifacts moved to reverse-DNS compliant Maven 2+ coordinates.
6.393 [INFO] 
6.393 [INFO] --- clean:3.2.0:clean (default-clean) @ live-client-api ---
6.400 [INFO] Deleting /root/live-client-api/target
6.405 [INFO] 
6.405 [INFO] --- apt:1.1.3:process (default) @ live-client-api ---
6.872 [INFO] 
6.872 [INFO] --- sentry:0.0.5:uploadSourceBundle (default) @ live-client-api ---
6.876 [WARNING] There's more than one source root, using /root/live-client-api/src/main/java
6.878 [INFO] Properties file not found
6.979 [INFO] Executing tasks
6.993 [INFO]      [exec] /bin/sh: 1: sentry-cli: not found
6.996 [INFO] ------------------------------------------------------------------------
6.996 [INFO] Reactor Summary for server 0.0.1-SNAPSHOT:
6.996 [INFO] 
6.996 [INFO] server ............................................. SUCCESS [  0.234 s]
6.996 [INFO] live-common ........................................ SUCCESS [  4.613 s]
6.996 [INFO] live-client-api .................................... FAILURE [  0.665 s]
6.996 [INFO] live-backend-api ................................... SKIPPED
6.996 [INFO] ------------------------------------------------------------------------
6.996 [INFO] BUILD FAILURE
6.996 [INFO] ------------------------------------------------------------------------
6.996 [INFO] Total time:  6.350 s
6.997 [INFO] Finished at: 2023-12-04T15:51:50Z
6.997 [INFO] ------------------------------------------------------------------------
6.997 [ERROR] Failed to execute goal io.sentry:sentry-maven-plugin:0.0.5:uploadSourceBundle (default) on project live-client-api: An Ant BuildException has occured: exec returned: 127
6.997 [ERROR] around Ant part ...<exec failOnError="true" executable="/bin/sh">... @ 4:51 in /root/live-client-api/target/antrun/build-main.xml
6.997 [ERROR] -> [Help 1]
6.997 [ERROR] 
6.997 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
6.997 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
6.997 [ERROR] 
6.997 [ERROR] For more information about the errors and possible solutions, please read the following articles:
6.997 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
6.997 [ERROR] 
6.997 [ERROR] After correcting the problems, you can resume the build with the command
6.997 [ERROR]   mvn <args> -rf :live-client-api
------
failed to solve: process "/bin/sh -c mvn clean install -Pprod -Dmaven.test.skip=true" did not complete successfully: exit code: 1
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,8 @@
 - Report dependencies ([#22](https://github.com/getsentry/sentry-maven-plugin/pull/22))
 - Send telemetry data for plugin usage ([#28](https://github.com/getsentry/sentry-maven-plugin/pull/28))
   - This will collect errors and timings of the plugin and its tasks (anonymized, except the sentry org id), so we can better understand how the plugin is performing. If you wish to opt-out of this behavior, set `<skipTelemetry>true</skipTelemetry>` in the sentry plugin configuration block.
+- Add `aarch64` sentry-cli ([#39](https://github.com/getsentry/sentry-maven-plugin/pull/39))
+  - This is used when the build is executed inside a docker container on an Apple silicon chip (e.g. M1)
 
 ### Dependencies
 

--- a/download-sentry-cli.sh
+++ b/download-sentry-cli.sh
@@ -10,7 +10,7 @@ function prop {
 base_url="$(prop 'repo' $props_file)/releases/download/$(prop 'version' $props_file)"
 target_dir="src/main/resources/bin/"
 actual_props_file="$target_dir${props_file}"
-PLATFORMS="Darwin-universal Linux-i686 Linux-x86_64 Windows-i686"
+PLATFORMS="Darwin-universal Linux-i686 Linux-x86_64 Windows-i686 Linux-aarch64"
 
 function shouldDownload {
     if ! cmp -s "$props_file" "$actual_props_file"; then

--- a/download-sentry-cli.sh
+++ b/download-sentry-cli.sh
@@ -10,7 +10,7 @@ function prop {
 base_url="$(prop 'repo' $props_file)/releases/download/$(prop 'version' $props_file)"
 target_dir="src/main/resources/bin/"
 actual_props_file="$target_dir${props_file}"
-PLATFORMS="Darwin-universal Linux-i686 Linux-x86_64 Windows-i686 Linux-aarch64"
+PLATFORMS="Darwin-universal Linux-i686 Linux-x86_64 Linux-aarch64 Windows-i686"
 
 function shouldDownload {
     if ! cmp -s "$props_file" "$actual_props_file"; then

--- a/src/main/java/io/sentry/SentryCliProvider.java
+++ b/src/main/java/io/sentry/SentryCliProvider.java
@@ -34,6 +34,7 @@ public class SentryCliProvider {
     final @Nullable String cliSuffix = getCliSuffix();
 
     if (cliSuffix != null && !cliSuffix.isBlank()) {
+      logger.info("Looking for CLI with suffix " + cliSuffix);
       final @NotNull String resourcePath = "/bin/sentry-cli-" + cliSuffix;
       final @Nullable String cliAbsolutePath = searchCliInResources(resourcePath);
 

